testsuite/integration/web/pom.xml

```diff
@@ -687,5 +687,487 @@
+                                        <arquillian.xml>arquillian-bootable.xml</arquillian.xml>
+                                    </systemPropertyVariables>
+                                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                                    <excludes>
+                                        <!-- requires legacy-security -->
+                                        <exclude>org/jboss/as/test/integration/web/security/**/*TestCase.java</exclude>
+                                        <!-- requires ee-concurrency -->
+                                        <exclude>org/jboss/as/test/integration/ee/concurrent/*TestCase.java</exclude>
+                                        <!-- Requires javax.validation.api module -->
+                                        <exclude>org/jboss/as/test/integration/web/annotationsmodule/WebModuleDeploymentTestCase.java</exclude>
+                                        <!-- requires https -->
+                                        <exclude>org/jboss/as/test/integration/web/handlers/RequestDumpingHandlerTestCase.java</exclude>
+                                    </excludes>
+                                    <classpathDependencyExcludes>
+                                        <classpathDependencyExclude>
+                                            org.wildfly.arquillian:wildfly-arquillian-container-managed
+                                        </classpathDependencyExclude>
+                                    </classpathDependencyExcludes>
+                                </configuration>
+                            </execution>
+
+                            <!-- Tests against the cloud-server install -->
+                            <!-- Reuse the microprofile.surefire execution as the valid tests are the same.
+                                 Just point to standalone.xml and the locally provisioned installation. -->
+                            <execution>
+                                <id>microprofile.surefire</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <environmentVariables>
+                                        <JBOSS_HOME>${jboss.dist}</JBOSS_HOME>
+                                    </environmentVariables>
+                                    <systemPropertyVariables>
+                                        <bootable.jar>${project.build.directory}/test-wildfly-cloud.jar</bootable.jar>
+                                        <arquillian.xml>arquillian-bootable.xml</arquillian.xml>
+                                        <jboss.server.config.file.name>standalone.xml</jboss.server.config.file.name>
```

Comment:
Is this appropriate for bootable JAR? I don't think there's a way to provide a configuration file...
Also lease check whether to fix the comment to this execution accordingly.

Reply:
Removed, thanks